registrations/views.py

```diff
@@ -162,3 +162,17 @@ def get(self, request, *args, **kwargs):
             }
         }
         return Response(resp, status=status)
+
+
+class ThirdPartyRegistrationView(APIView):
+
+    """ ThirdPartyRegistrationView Interaction
+        POST - starts up the task that pulls registrations from a 3rd party
+    """
+    permission_classes = (IsAuthenticated,)
+
+    def post(self, request, *args, **kwargs):
+        status = 200
```

Comment:
Maybe a better code is `201`? Since we're telling the client that we created the task?